index.html

```diff
@@ -195,7 +195,7 @@ <h2 class="big-picture__title  visually-hidden">Просмотр фотогра
 
   <!-- Шаблон изображения случайного пользователя -->
   <template id="picture">
-    <a href="#" class="picture">
+    <a href="#" class="picture" data-id="">
```

Comment:
```suggestion
    <a href="#" class="picture">
```
не нужно добавлять по умолчанию